wiki/howToUseSemantics.md

```diff
@@ -0,0 +1,42 @@
+## How to use semantics
+
+To import the project user need to add to pom.xml
+
+    <repositories>
+       <repository>
+           <id>jitpack.io</id>
+           <url>https://jitpack.io</url>
+       </repository>
+    </repositories>
+    <dependency>
+        <groupId>com.github.eiffel-community</groupId>
+        <artifactId>eiffel-remrem-semantics</artifactId>
+        <version>x.x.x</version>
+    </dependency>
+
+Or import the project to pom.xml
```

Comment:
not in pom.xml , I think you are refering to build file in groovy?

Reply:
This code snippet is used in the Gradle script and mentioned the same in the document as well.